Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -0,0 +1,169 @@
+# Detailed overview of ArangoSearch views
+
+ArangoSearch provides a fulltext search and it is only a subset of its available 
```

Comment:
Still not sure how to read this sentence.

> it is only a subset of its available functionality

Should it state that ArangoSearch is more than fulltext search? Or does `its` refer to IResearch (i.e. ArangoSearch provides only certain things that IResearch is capable of)?

Reply:
Reformulated.